lib/all_futures/persist.rb

```diff
@@ -30,7 +30,6 @@ def delete
 
     def destroy
       _raise_readonly_record_error if readonly?
-      @_trigger_destroy_callback = persisted? ? _delete_record > 0 : true
```

Comment:
In your note, you expressed confusion about the `destroy` method not deleting the record. I would offer that it used to delete, but you might have been over-ambitious in your code removal. :heart:

The ivar got set based on Redis returning a `1` to indicate that a record was actually deleted.

As I said earlier, hopefully and perhaps we don't need these ivars; in this case, we should replace this line with a simple call to `_delete_record`.

Reply:
:doh: I read `_delete_record` as an integer value getter not a state mutation method that then returns the truth or falsity of its success as an integer. The fact that `destroy` deleted the record in Redis makes the broken test make sense and we can indeed fix it as you say.

Reply:
Simply have `destroy` call `delete` after the `_raise_readonly_record_error` line now